examples/timer-log-xml/README.adoc

```diff
@@ -0,0 +1,28 @@
+= timer-log-xml
+
+This is a basic hello world example that use XML to set-up
+a Camel timer that triggers every second and prints to the
+log.
+
+To run it:
+
+[source,text]
+----
+$ mvn clean compile quarkus:dev -DnoDeps -Dcamel.main.xml-routes=file:$PWD/src/main/resources/routes/my-routes.xml
```

Comment:
`$PWD` won't work on Windows. Does the `$PWD` have to be there actually? It would be nice if `file:src/main/resources/routes/my-routes.xml` worked too.

Reply:
It appears not to be working in dev love, need to digg into this a little bit more and I’ll follow up with a new PR